google-map.html

```diff
@@ -44,6 +44,8 @@
         start-address="San Francisco" end-address="Mountain View">
     </google-map-directions>
 
+<b>Note</b> You can disable dragging by adding `draggable="false"` on the `google-map` element.
```

Comment:
I double checked and it is indeed map: https://github.com/AreaFiftyLAN/lancie-frontend/commit/82a3f16bbc790b1e619348afcd46a5617d6b7e3b#diff-56474487828a2a7dd43e6e208d9fbe5cR124